include/dlt/dlt_common.h

```diff
@@ -88,20 +89,13 @@
 #      include <time.h>
 #   endif
 
-#   if defined(__GNUC__)
```

Comment:
Changes to dlt_common.h (and all other header files in this folder) should be treated as API changes. Therefore this patch would require another major version release of DLT.

is there a way to keep the header files untouched for this feature?

Reply:
Hi @michael-methner ,
I don't think we can do this feature completely untouched. But we made sure that we are 100% compatible with the old header (otherwise we would have had various build problems in projects using our DLT header).
As requested by the other comment, I would remove the movement from dlt_common.h to dlt_log.h. I hope then the compatibility becomes more visible.